lms/static/js/demographics_collection/DemographicsCollectionModal.jsx

```diff
@@ -326,84 +326,92 @@ class DemographicsCollectionModal extends React.Component {
                     }}
                   />
                   {/* Family Income */}
-                  <label htmlFor={FIELD_NAMES.INCOME}>
-                    {gettext("What was the total combined income, during the last 12 months, of all members of your family? ")}
-                  </label>
-                  <select
-                    onChange={this.handleSelectChange}
-                    className="form-control"
-                    name={FIELD_NAMES.INCOME} id={FIELD_NAMES.INCOME}
-                    value={wizardConsumer[FIELD_NAMES.INCOME]}
-                    disabled={this.state.fieldError}
-                  >
-                    <option value="default">{gettext("Select income")}</option>
-                    {
-                      this.loadOptions(FIELD_NAMES.INCOME)
-                    }
-                  </select>
+                  <div className="d-flex flex-column pb-3">
```

Comment:
Most of the changes in this file are from me wrapping the `label` & `select` question-pairs into `divs` to apply styling the desired styling classes.